lib/pbench/test/unit/server/conftest.py

```diff
@@ -1,4 +1,5 @@
 import datetime
+from http import HTTPStatus
 import os
 import pytest
 import shutil
```

Comment:
I think we are supposed to be grouping the bare `import`'s together and the `from` imports together, not interspersing them.  (So, line 2 should move to between lines 6 and 7.)